cicecore/cicedynB/dynamics/ice_dyn_shared.F90

```diff
@@ -946,31 +949,44 @@ subroutine seabed_stress_factor_LKD (nx_block, ny_block,         &
          hwater      ! water depth at tracer location (m)
 
       real (kind=dbl_kind), dimension (nx_block,ny_block), intent(inout) :: &
-         Tbu         ! seabed stress factor (N/m^2)
+         Tbu         ! seabed stress factor at 'grid_loc' (N/m^2)
+
+      character(len=*), optional, intent(inout) :: &
+         grid_location    ! grid location (U, E, N), U assumed if not present
 
       real (kind=dbl_kind) :: &
-         au,  & ! concentration of ice at u location
-         hu,  & ! volume per unit area of ice at u location (mean thickness, m)
-         hwu, & ! water depth at u location (m)
-         hcu    ! critical thickness at u location (m)
+         au,  & ! concentration of ice at 'grid_loc' location
```

Comment:
I would simplify the comments as 
!     ... at grid_location

Reply:
thanks, fixed.